src/olympia/scanners/migrations/0038_scannerqueryrule_completed.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2020-09-29 15:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('scanners', '0037_auto_20200717_1233'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='scannerqueryrule',
+            name='completed',
+            field=models.DateTimeField(blank=True, default=None, null=True),
+        ),
+    ]
```

Comment:
We handle `None` just in case where we display it, so I think it's fine to leave it empty for past results where we don't know the actual completion datetime.